editor/src/clj/editor/app_view.clj

```diff
@@ -2397,7 +2398,8 @@ If you do not specifically require different script states, consider changing th
     (let [game-project (project/get-resource-node project "/game.project" evaluation-context)
           package (game-project/get-setting game-project ["android" "package"] evaluation-context)
           project-title (game-project/get-setting game-project ["project" "title"] evaluation-context)
-          apk-path (io/file output-directory project-title (str project-title ".apk"))]
+          binary-name (BundleHelper/projectNameToBinaryName project-title)
+          apk-path (io/file output-directory binary-name (str binary-name ".apk"))]
```

Comment:
This is new 1.8.1 change https://github.com/defold/defold/pull/8660
Shouldn't be merged until release of the 1.8.1